spec/requests/v6/assessments_controller/realistic_civil_apply_spec.rb

```diff
@@ -0,0 +1,244 @@
+require "rails_helper"
+
+# Purpose of this spec is to test the real submission by the "Civil Apply" covering as many fields filled in as possible
+# Civil apply never send data for employment_details, self_employment_details and partner
+
+module V6
+  RSpec.describe AssessmentsController, :calls_bank_holiday, type: :request do
+    describe "POST /create" do
+      before do
+        create :state_benefit_type, label: "personal_independent_payments"
+        post v6_assessments_path, params: params.to_json, headers:
+      end
+
+      let(:headers) { { "CONTENT_TYPE" => "application/json", "Accept" => "application/json", 'HTTP_USER_AGENT': "CivilApply/1.0 production" } }
+      let(:params) do
+        {
+          "assessment": {
+            "level_of_help": "certificated",
+            "submission_date": "2023-08-15",
+            "client_reference_id": "3000-01-01",
```

Comment:
This looks like a date. Please use the sorts of IDs we get from Apply, including letters